File: sdkwork-ai-sdk-kotlin/src/main/kotlin/com/sdkwork/ai/ChatCompletionResponse.kt
package com.sdkwork.ai

data class ChatCompletionResponse(
    val id: String? = null,
    val object: String? = null,
    val created: Int? = null,
    val model: String? = null,
    val choices: List<Choice>? = null,
    val usage: Usage? = null,
    val systemFingerprint: String? = null,
    val serviceTier: String? = null
)

File: sdkwork-ai-sdk-typescript/src/types/chat-completion-response.ts
import type { Choice } from './choice';
import type { Usage } from './usage';

/** 聊天完成响应 */
export interface ChatCompletionResponse {
  /** 请求唯一标识 */
  id?: string;
  /** 对象类型 */
  object?: string;
  /** 创建时间（Unix时间戳，秒） */
  created?: number;
  /** 使用的模型名称 */
  model?: string;
  /** 模型输出内容列表 */
  choices?: Choice[];
  /** token用量 */
  usage?: Usage;
  /** 系统指纹 */
  system_fingerprint?: string;
  /** 服务等级 */
  service_tier?: string;
}
